src/launchpad/actions/expressions.js

```diff
@@ -0,0 +1,57 @@
+const constants = require("../constants");
+const { generateKey } = require("../utils/utils");
+
+function evaluateInput(input) {
+  return async function ({dispatch, client}) {
+    if (!client) {
+      console.warn("No client");
+      return;
+    }
+
+    try {
+      const packet = await client.clientCommands.evaluate(input, {});
+      dispatch(addExpression(input, packet));
+    } catch (err) {
+      console.warn("Error when evaluating expression", err)
+    }
```

Comment:
I prefer to be paranoid here, since it's a promise, it can be rejected (by some changes server side or whatever), and I like to have a safety net that tells us what's going wrong